tests/utils.go

```diff
@@ -3666,6 +3669,19 @@ func HasCDI() bool {
 	return HasFeature("DataVolumes")
 }
 
+func HasCeph() (bool, string) {
```

Comment:
I'm just wondering. Is this only a ceph block storage?
Or can ceph be filesystem as well? If it can, how do we distinguish here?

Reply:
Specifically on kubevirtci, seems that at the moment [only ceph block storage may be deployed](https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/k8s/scripts/ceph-csi.sh#L16). That's what I see also in a cluster in which `enable-ceph` is set

Reply:
Changed `GetCephStorageClass` and added tests for both Ceph filesystem and block volumes in vm/console tests